src/test/java/org/broadinstitute/hellbender/tools/spark/sv/utils/SVFastqUtilsUnitTest.java

```diff
@@ -29,7 +30,7 @@
 /**
  * Unit tests for components in {@link SVFastqUtils}.
  */
-public class SVFastqUtilsUnitTest {
+public class SVFastqUtilsUnitTest extends BaseTest {
```

Comment:
After https://github.com/broadinstitute/gatk/pull/3475, this should extend `GATKBaseTest`